Source/WebKit/UIProcess/Extensions/Cocoa/_WKWebExtensionStorageSQLiteStore.mm

```diff
@@ -71,14 +71,34 @@ - (instancetype)initWithUniqueIdentifier:(NSString *)uniqueIdentifier storageTyp
     return self;
 }
 
+- (void)getAllKeys:(void (^)(NSArray<NSString *> *keys, NSString *errorMessage))completionHandler
+{
+    auto weakSelf = WeakObjCPtr<_WKWebExtensionStorageSQLiteStore> { self };
+    dispatch_async(_databaseQueue, ^{
+        auto strongSelf = weakSelf.get();
```

Comment:
Not new to this, but the pointer should be noted here (and other places in this file.)

```suggestion
        auto *strongSelf = weakSelf.get();
```

Reply:
Ah okay, so the type of `strongSelf` (`RetainPtr`) has implicit conversion operators to the underlying pointer?

Reply:
Ah I forgot `weakSelf.get()` returned a `RetainPtr`. Just keep this as plain `auto` still then.

Reply:
Or this needs an extra `get()`:

```suggestion
        auto *strongSelf = weakSelf.get().get()`
```

Reply:
Depends, do you think having the pointer notation is worth having a `.get().get()`? I might update it across other files as well.

Reply:
If it is in extension code, I think ObjC pointer is better, since we use ARC in our files. Otherwise `RetainPtr` is needed for all other uses in WebKit since they don't use ARC.

Reply:
Made an in-PR commit for this ✅